.github/workflows/dispatch-workflow.yml

```diff
@@ -57,3 +57,12 @@ jobs:
     secrets: inherit
     with:
       profile: ${{ inputs.profile }}
+
+  ohos:
+    if: ${{ inputs.workflow == 'ohos' }}
+    name: OpenHarmony
+    uses: ./.github/workflows/ohos.yml
+    # I believe we don't need any secrets in the OpenHarmony CI workflow.
```

Comment:
Perhaps we can remove the commented lines.

Reply:
done